componentsCss/slider.css

```diff
@@ -0,0 +1,41 @@
+@import '../style.css';
+
+.slidecontainer {
+    width: 100%;
+    margin: 10px;
+    padding: 10px;
+  }
+  
+  .slider {
+    -webkit-appearance: none;
+    width: 80%;
+    height: 15px;
+    border-radius: 5px;
+    background: var(--darkgray-color);
+    outline: none;
+    opacity: 0.7;
+    -webkit-transition: .2s;
+    transition: opacity .2s;
+  }
+  
+  .slider:hover {
+    opacity: 1;
+  }
```

Comment:
try to use opacity with colors, if possible

Reply:
ok